.github/workflows/dependency-review.yml

```
@@ -62,6 +62,6 @@ jobs:
         uses: actions/dependency-review-action@2031cfc080254a8a887f58cffee85186f0e49e48 # v4
         with:
           comment-summary-in-pr: always
-        #   fail-on-severity: moderate
-        #   deny-licenses: GPL-1.0-or-later, LGPL-2.0-or-later
+        fail-on-severity: moderate
```

actionlint (1.7.12)

[error] unexpected key "fail-on-severity" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)